adapters/tlsio_wolfssl.c

```diff
@@ -999,6 +1010,12 @@ int tlsio_wolfssl_setoption(CONCRETE_IO_HANDLE tls_io, const char* optionName, c
             tls_io_instance->ignore_host_name_check = *server_name_check;
             result = 0;
         }
+        else if (strcmp("debugLog", optionName) == 0)
+        {
```

Comment:
is this not checking the `value`. What if it's `false`?

Reply:
Good point: we need to check non-zero. Will add.